DEVELOPING.md

```diff
@@ -0,0 +1,61 @@
+## Logging
+
+How to log in MCS:
+
+```bash
+std::cout << row.toString() << std::endl;
```

Comment:
I think it would be good to mention that both Row and RowGroup have ::toString() method. Not sure about the actual phrasing though.